lib/factory_trace/processors/find_unused.rb

```diff
@@ -30,88 +32,89 @@ def self.call(defined, used)
         output
       end
 
-      private
+      class << self
+        private
```

Comment:
**Note:** You can put above `self.call` to be in inside the class too.

Reply:
Fixed a8d965f2c138f5db7e2754b676269b1d6bb983b8